includes/class-boldgrid-inspirations-purchase-for-publish.php

```diff
@@ -1005,12 +1005,35 @@ public function get_post_status() {
 		*/
 		$post_status = apply_filters( 'boldgrid_cart_post_status', $post_status );
 
-		// Implode our array of $post_status to be used alongside an IN statement.
-		$post_status = '( "' . implode( '", "', esc_sql( $post_status ) ) . '" )';
-
 		return $post_status;
 	}
 
+	/**
+	 * Generate prepare placeholders
+	 *
+	 * When properly preparing a mysql statement, the array
+	 * string is dynamically generated to prevent improper
+	 * escaping. This method will generate the array of placeholders
+	 * to be used by that prepare string.
+	 *
+	 * @param array $values,
+	 * @return array $placeholders
+	 */
+	public function get_prepare_placeholders( $values ) {
```

Comment:
This method is get `prepare` `placeholders`.

The method itself is not doing any sql preparing, nor is it doing anything with placeholders. It's simply looping through an array and merging things.

Is it really needed? Maybe we rename it to something that makes more sense?